apps/slack/slack.py

```diff
@@ -13,3 +14,9 @@
 os: mac
 and app.bundle: com.tinyspeck.slackmacgap
 """
+
+@ctx.action_class("user")
+class UserActions:
```

Comment:
@rntz, I think that this is actually applying outside the Slack context. Any tips on how to make this override only apply to Slack?

Reply:
```py
ctx.matches = r"""
app: slack
"""
```

Reply:
Ah, the context needs to be defined as well as the mod? Makes sense - thank you